client/packages/system/src/Item/api/operations.graphql

```diff
@@ -139,11 +139,14 @@ query itemsWithStats(
   $key: ItemSortFieldInput
   $isDesc: Boolean
   $filter: ItemFilterInput
+  $first: Int
+  $offset: Int
 ) {
   items(
     storeId: $storeId
     sort: { key: $key, desc: $isDesc }
     filter: $filter
+    page: { first: $first, offset: $offset }
```

Comment:
that wasn't needed for this PR or did I miss it?

Reply:
sorry, offroad, sort of. The list of items shown was being truncated by the server default pagination limit, even though the client was at some point specifying a much higher limit. This meant on the datafile being used, only items up to `c` were shown